doc/Install.md

```diff
@@ -162,5 +162,5 @@ Bad things will happen.
 Let there be a hardware failure, a corrupted database or whatever you can think of.
 So once the installation of your Friendica node is done, you should make yourself a backup plan.
 
-The most important file is the `.htconfig.php` file in the base directory.
+The most important file is the `config/local.ini.php` file in the base directory.
```

Comment:
The config file seems not to be any longer in the base directory. Could you change this.